src/cdk-experimental/menu/menu-bar.ts

```diff
@@ -20,18 +35,169 @@ import {CDK_MENU, Menu} from './menu-interface';
   selector: '[cdkMenuBar]',
   exportAs: 'cdkMenuBar',
   host: {
+    '(keydown)': '_handleKeyEvent($event)',
+    '(focus)': 'focusFirstItem()',
```

Comment:
If we're redirecting focus anyway, couldn't we remove the `tabindex` from this this and let the user tab into the first item directly?

Reply:
yeah, I didn't want to pollute this PR with too much tab focus logic. I'll have a follow on which sets the tabindex of the first menu item in the menu bar to 0 and handles `changes` as well.

Reply:
Considering this further, the current implementation will set focus on the first non-disabled item and keeps focus on the MenuBar if all elements are disabled which is the expected behaviour. Any changes would essentially do the same. The only difference being that the tab index of the first menu item would be 0 rather than -1 as is here. Thoughts?